src/compiler.rs

```diff
@@ -35,12 +34,12 @@ impl AmberCompiler {
             path,
             cli_opts,
         }
-        .load_code(AmberCompiler::strip_off_shebang(code))
+        .load_code(AmberCompiler::comment_shebang(code))
     }
 
-    fn strip_off_shebang(code: String) -> String {
+    fn comment_shebang(code: String) -> String {
         if code.starts_with("#!") {
-            code.split('\n').skip(1).collect_vec().join("\n")
+            String::from("// ") + &code
```

Comment:
Actually... can we replace shebang with an empty line? The comments can now persist in the generated bash file: https://github.com/amber-lang/amber/pull/403

Reply:
We could, but wouldn't it be better to show that there was a shebang in the input file, for debugging purposes?

Reply:
Okay let's leave it